### DISCUSSION TITLE
Too many documentation kill the documentation ?

### DISCUSSION THREAD

**Discussion by Carreau** [jupyter/jupyter#745] [Ideas]
Antoine de Saint Exupery said:

> It seems that perfection is attained not when there is nothing more to add, but when there is nothing more to remove.

And honestly the first page of our documentation page is starting to scare me: 
- 26 links on the side TOC, 
- I cannot make a distinction between some sections between many section. 

Example: 

  – Jupyter Notebook Quickstart – Seem great. 
  – User Stories – Why ? What does that mean, why do I care ? 
  – Projects – Hum fair enough
  – Documentation by project – Ok, what's the difference with above ?
  – Technical overview – Great if I want to contribute
  – Contributor Guide – Ah, no wait, this is where I want to contribute
  –  IPython Development guide – Is that what I should click on to contribute ? 

Wait, this are just link that scroll the page 10 lines, the first 4 of which I can already see on my screen ?

Oh, and If I remove it and the next section that also just provide link to anchor on the same page, I can  have the full page on my screen at once. 

Ah, and then I realise that this TOC is exactly he same that the sidebar. 

So can we try to **decrease** the amount of choice a use have when landing on the documentation page, not increase it. 

As a **user** when i see such an introduction page, I am running away.

**Comment by willingc** [comment]
Point taken.

**Comment by Carreau** [comment]
> Point taken.

Hope you did not take that against you, you are doing a **fantastic** job, but the recent discussion might have steered us away by trying to directly tackle a long term goal. 

And it's definitively far from easy with all the scope of Jupyter.
